src/adaptors/utils.js

```diff
@@ -22,7 +22,7 @@ exports.formatChain = (chain) => {
   )
     return 'zkSync Era';
   if (chain && chain.toLowerCase() === 'polygon_zkevm') return 'Polygon zkEVM';
-  return chain.charAt(0).toUpperCase() + chain.slice(1);
+  return chain.charAt(0).toUpperCase() + chain.slice(1).toLowerCase();
```

Comment:
Fixed the function to return the chain name in sentence case

Reply:
@Addis-Belete We shouldn't update these functions, only our folder